woocommerce/wc-admin
# Remove business step original flow

Fixes #6977

This PR removes:
- The original business step flow
- Benefits step
- Related e2e tests

### Detailed test instructions:

1. Disable `remote-extensions-list` feature in `config/development.json`, or run `npm start --features '{"remote-extensions-list":false}'`.
2. Go to `/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard&step=store-details`.
3. Set up store address as Afghanistan - previously, this will apply the original flow.
2. Continue setup until the Business Detail step.
3. Observe that it displays the new flow.
4. Additionally, make sure everything else works as expected.

### Screenshot

![image](https://user-images.githubusercontent.com/3747241/120461877-fc62a500-c3cc-11eb-8d5b-194a485f1663.png)
